docs/authentication/saml/azure.mdx

```diff
@@ -56,14 +57,24 @@ description: Learn how to allow users to sign into your Clerk app with their Mic
   1. Select the **Select** button at the bottom of the page.
   1. Select the **Assign** button at the bottom of the page.
 
-  ### Configure single sign-on (SSO) in Azure
+  ### Configure single sign-on (SSO) in Microsoft
 
   After you have assigned the user or group of users to the enterprise application, you need to configure the single sign-on (SSO) settings to enable SAML SSO.
 
-  1. In the navigation sidebar, select **Overview**.
-  1. Under the _Getting started_ section, select **Set up single sign-on**.
+  1. In the navigation sidebar, select **Manage** and select **Single sign-on**.
   1. Select **SAML** as the single sign-on method.
 
+  ### Configure your identity provider
+
+  To configure your identity provider (Microsoft Entra ID), you need to add the following fields to your application:
+
+  1. Find the **SAML Certificates** section.
+  1. Copy the **App Federation Metadata Url**.
```

Comment:
A bit of a nit, but let's be consistent with the casing for URL, as we have below, eg: Assertion Consumer Service URL:
```suggestion
  1. Copy the **App Federation Metadata URL**.
```